.gitlab/deploy_packages/oci.yml

```diff
@@ -21,6 +21,9 @@ include:
     - export VERSION=$(inv agent.version --url-safe)-1
     - git config --global credential.helper '!f() { echo username=x-access-token; echo "password=$GH_TOKEN"; };f'
     - ${CI_PROJECT_DIR}/tools/ci/retry.sh git clone -v --depth=1 https://github.com/DataDog/datadog-packages /tmp/datadog-packages
+    - git config --global url."https://gitlab-ci-token:${CI_JOB_TOKEN}@gitlab.ddbuild.io/DataDog/".insteadOf "https://github.com/DataDog/"
```

Comment:
Could we do it before in order to clone `datadog-packages` from gitlab as well?

Reply:
It fails for some reason, probably a permission issue. We can always fix it in another PR